odd_collector/adapters/redshift/adapter.py

```diff
@@ -65,7 +68,7 @@ def get_data_entities(self) -> List[DataEntity]:
                 f"Load {len(mtables.items)} Datasets DataEntities from database"
             )
 
-            return map_table(self.__oddrn_generator, mtables, mcolumns, self.__database)
+            return map_table(self.__oddrn_generator, mtables, mcolumns, self.__database, self.__schemas)
```

Comment:
This way you still fetch everything and just map included schemas. It'd actually be better if you didn't fetch schemas you don't need to